code/midtown/mmai/aiGoalFollowWayPts.cpp

```diff
@@ -55,3 +57,124 @@ b32 aiGoalFollowWayPts::Context()
         return false;
     }
 }
+
+i32 aiGoalFollowWayPts::GetWayPtId(i16 index, i32 num_waypts)
```

Comment:
Was there a reason for passing in `num_waypts` instead of using `NumWayPts` directly?

Reply:
No, it was an oversight. I changed it now